File: app/src/main/java/com/purpletealabs/bitcoinapp/dtos/Bpi.java
package com.purpletealabs.bitcoinapp.dtos;

import com.google.gson.annotations.SerializedName;

public class Bpi {
    @SerializedName("USD")
    private Price usdPrice;

    @SerializedName("GBP")
    private Price gbpPrice;

    @SerializedName("EUR")
    private Price eurPrice;

    public Price getUsdPrice() {
        return usdPrice;
    }

    public Price getGbpPrice() {
        return gbpPrice;
    }

    public Price getEurPrice() {
        return eurPrice;
    }
}

File: app/src/main/java/com/purpletealabs/bitcoinapp/dtos/Bpi.kt
package com.purpletealabs.bitcoinapp.dtos

import com.google.gson.annotations.SerializedName

class Bpi {
    @SerializedName("USD")
    val usdPrice: Price? = null

    @SerializedName("GBP")
    val gbpPrice: Price? = null

    @SerializedName("EUR")
    val eurPrice: Price? = null
}
